examples/3d/spotlight.rs

```diff
@@ -48,6 +48,8 @@ fn setup(
     ));
 
     // cubes
+
+    // Make it play out the same way every time with deterministic results. We do this for testing purposes.
```

Comment:
```suggestion
    // Make it deterministic for testing purposes.
```

I personally don't like "make it play out" as a phrase. I feel like determinism is a common enough term, or at least easily googled, so it should be enough to just say that.

Reply:
Done!